src/expr/core/src/aggregate/user_defined.rs

```diff
@@ -42,8 +42,11 @@ impl AggregateFunction for UserDefinedAggregateFunction {
 
     /// Creates an initial state of the aggregate function.
     fn create_state(&self) -> Result<AggregateState> {
-        let state = self.runtime.call_agg_create_state()?;
-        Ok(AggregateState::Any(Box::new(State(state))))
+        // FIXME(eric): This is bad. Let's make `create_state` async if someday we allow async UDAF
+        futures::executor::block_on(async {
```

Comment:
Is `spawn_blocking` acceptable here?

Reply:
It's not in used yet, because the syntax `create aggregate with (async = true)` is not supported yet.